lib/VerticalDelegate.js

```diff
@@ -408,7 +408,8 @@ module.exports = {
 		if (!list.hasReset) return this.reset(list);
 
 		var cpp = this.controlsPerPage(list),
-			end = Math.max(list.$.page1.start, list.$.page2.start) + cpp;
+			end = Math.max(list.$.page1.start, list.$.page2.start) + cpp - 1,
```

Comment:
Good point. 
{{end}} means last index of pages, so we should subtract 1.
Thank you @anishlg